docs/manual/docs/cmd/spo/field/field-remove.md

```diff
@@ -19,7 +19,7 @@ Option|Description
 `--listUrl [listUrl]`|Server- or web-relative URL of the list where the field is located. Specify only one of `listTitle`, `listId` or `listUrl`
 `-i, --id [id]`|The ID of the field to remove. Specify `id` or `fieldTitle` but not both
 `--fieldTitle [fieldTitle]`|The display name (case-sensitive) of the field to remove. Specify `id` or `fieldTitle` but not both
-`--confirm`|Don't prompt for confirming removing the field
+`--confirm`|Don't prompt for confirmation before removing the user from the group
```

Comment:
Could it be that you've changed this by accident (file command referring to confirming removing user from group)

Reply:
Fixed it in a second pull request